Codeinwp/neve
# chore: add header components popup upsell

### Summary
-  adds components upsell inside the header builder components inserter popover
- refactors the components popover to move it into its' own file


### Will affect visual aspect of the product

NO

### Test instructions


- Launch an instance without the pro version; 
- Go into *Customizer > Header* and click on any of the slots' plus button;
- Inside the popover there should be some locked components;
- Below that, there should be a CTA to the themeisle website;
- With PRO version enabled, the pro components should be separated from the free ones in all three builders;



Closes Codeinwp/neve-pro-addon#1724.
